cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceEnsurer.java

```diff
@@ -52,7 +53,11 @@ protected void doStart() {
               // if there is no exception, assume successfully created and break
               LOG.info("Successfully created namespace '{}'.", NamespaceMeta.DEFAULT);
               notifyStarted();
-            } catch (AlreadyExistsException e) {
+            } catch (FileAlreadyExistsException e) {
+              LOG.warn("Got exception while trying to create namespace '{}'.", NamespaceMeta.DEFAULT, e);
```

Comment:
FileAlreadyExistsException would only happen if the namespace is not in app.meta, but we try to create the namespace directory (and it happens to already exist).
So its not that the namespace already exists, but the files for it on hdfs already exist.